etc/initdev/Vagrantfile

```diff
@@ -4,7 +4,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "ubuntu/vivid64" # 15.04
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
Me, either. I thought it was the dev environment this morning, but then figured out the docker flow later on the slack. Will try the host environment later.